Core/src/main/java/put/io/black/java/core/logic/FileManager.java

```diff
@@ -0,0 +1,73 @@
+package put.io.black.java.core.logic;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileManager {
+
+    public static final String PATH = "Scenarios/";
+    public static final String FILE_NOT_EXIST = "File not exist";
+
+    private boolean ready;
+
+    public FileManager() {
+        new File(PATH).mkdir();
```

Comment:
I will prepare this as check exists correct path and try make if not.

Reply:
Repaired